docs/content/SegmentedControl.mdx

```diff
@@ -1,14 +1,15 @@
 ---
+componentId: segmentedControl
 title: SegmentedControl
-status: Draft
 description: Use a segmented control to let users select an option from a short list and immediately apply the selection
+status: Alpha
```

Comment:
The component lifecycle label doesn't appear to be updating. Is there something else I need to do to get this showing "Alpha" instead of "Drafts"?

![Screen Shot 2022-08-15 at 11 56 22 AM](https://user-images.githubusercontent.com/2313998/184670297-13c50f17-48b9-4af9-ad03-96980c133160.png)

Reply:
This might be related to staging build because I see your changes on my machine 🤔

Reply:
Deployment is fixed now, turns out it was my fault ✨ 🙃

Navigation through arrow keys still doesn't work for me, but otherwise looks perfect!